src/pseudopeople/noise_functions.py

```diff
@@ -164,20 +164,17 @@ def miswrite_numerics(
     returns: pd.Series with some numeric values experiencing noise.
     """
 
-    token_noise_level = configuration.token_noise_level
+    token_noise_level = (
+        configuration.token_noise_level * 1 / 0.9
+    )  # This is fix to not replace with the original token
```

Comment:
Nit: Don't use same line comments like this, since black borks the formatting. Also no need to multiply by 1 :)

Instead do 
```
# This is fix to not replace with the original token
token_noise_level = configuration.token_noise_level / 0.9
```